tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -499,10 +499,18 @@ TEXT_DRAW_TEST_P(Fonts, TestFonts) {
     __DrawLoremIpsum(context, path.get(), keys, values);
 }
 
+// Segoe UI is noticeably different on ARM, but we still want to test other fonts so only remove Segoe
+#ifdef WINOBJC_DISABLE_TESTS
```

Comment:
For this one we could simply get rid of the Segoe test completely since we already test other fonts.

This might be the only place CTFontDrawGlyphs is validated, so that one shouldn't be removed, 
maybe just make that one a different font and hope x86 and ARM don't diverge on that one.

As for drawing with WinObjC-Regular I don't believe we have implemented setting different fall-back fonts so we're at the mercy of the platform.

I'm not sure if we should expect the fonts to converge at any point or if this is just a difference we're going to have to live with.  Thoughts @DHowett-MSFT @rajsesh-msft?